canonical-websites/www.ubuntu.com
# UA for Infrastructure - Update /support/plans-and-pricing page

## Done

- Updated the plans-and-pricing page for UA for infrastructure
- Updated the includes that appear in openstack and kubernetes that relate to this page
- Removed some unused includes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare to the [copy doc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#)

## Issue / Card

Fixes #4704
